test/Test_CarbonVote.js

```diff
@@ -1,46 +1,61 @@
+// accounts setting in this test environment:
+// by the nature of DSAuth and DSGuard, accounts[0] has root power. 
```

Comment:
move those comments to  the block of async (accounts) with the declared account constants